plugins/outputs/wavefront/wavefront.go

```diff
@@ -77,30 +78,50 @@ func (*Wavefront) SampleConfig() string {
 	return sampleConfig
 }
 
+func SenderURLFromURLAndToken(rawURL, token string) (string, error) {
+	newURL, err := url.Parse(rawURL)
+	if err != nil {
+		return "", fmt.Errorf("could not parse the provided Url: %s", rawURL)
+	}
+	newURL.User = url.User(token)
+
+	return newURL.String(), nil
+}
+
+func SenderURLFromHostAndPort(host string, port int) string {
```

Comment:
Can we please unexport these two functions as they are not used elsewhere!?

Reply:
Thanks! I will make that change on my push.